release-build.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+function showUsage
+{
+  echo -e "\nThis script is used to build a release"
+  echo
+  echo -e "Usage: ./`basename $0 $1 $2` actual_version next_version previous_version"
+  echo
+  echo -e "Example to build release 4.4.0 with next version 4.4.1 and previous version 4.2.5:"
+  echo -e "\t./`basename $0 $1` 4.4.0 4.4.1 4.2.5"
+  echo
+}
+
+if [ "$1" = "-h" ]
+then
+	showUsage
+	exit
+fi
+
+if [ $# -ne 3 ]
+then
+  showUsage
+  exit
+fi
+
```

Comment:
May want to consider checking for the version format that was supplied?

```
if ! [[ $1 =~ ^[0-9.]+$ ]] && [[ $2 =~ ^[0-9.]+$ ]] && [[ $3 =~ ^[0-9.]+$ ]]
then
    echo "Invalid version format supplied in one of the parameters."
	  showUsage
    exit
fi
```

This regular expression only checks for numbers and decimal.  You could elaborate on the expression if needed.

Reply:
yes about the version arguments, discussing with @josegar74 yesterday maybe the script should get the version from the pom (and we can suppose the version in the pom is the good one) and then build the release for the current branch we are in - but ideas and comments welcomed. Making the release yesterday we were raising some questions here and there and having a script which does all  automatically that we can be confident with would be super!